framework/CodeInterfaces/CobraTF/CTFparser.py

```diff
@@ -0,0 +1,96 @@
+"""
+Created on March 28, 2018
+@author: JYoo
+"""
+import os
+import fileinput
+import re
+
+class CTFparser():
+   """
+    Import the CobraTF input as list of lines, provide methods to add/change entries and print it back
+   """
+   def __init__(self,inputFile):
+	"""
+	# Constructor
+        # @ In, inputFile, string, input file name
+        # @ Out, None
+	"""
+        self.printTag = 'CTF PARSER'
+        if not os.path.exists(inputFile):
+            raise IOError(self.printTag + ' ERROR: not found CTF input file')
+
+        IOfile = open(inputFile, 'r')
+        self.inputfile = inputFile
+        """# modified list in a form of Dictionary"""
```

Comment:
```"""``` not ok in the body of the source code. This is used for doctrings only. Please use ```#``` for comments in the code